src/main/resources/mappers/lt/velykis/roberta/budget/app/account/transaction.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="lt.velykis.roberta.budget.app.transaction.TransactionRepository">
+    <!--    result maps     -->
+
+    <resultMap id="transactionMap" type="lt.velykis.roberta.budget.app.transaction.Transaction">
+        <id column="id" property="id"/>
+        <result column="accountId" property="accountId"/>
+        <result column="date" property="date"/>
+        <result column="description" property="description"/>
+        <result column="amount" property="amount"/>
+        <association property="account" javaType="lt.velykis.roberta.budget.app.account.Account">
```

Comment:
I think there is a way to reuse the mapping from another file - you should be able to reference it somehow using the namespace

Reply:
try `<association property="account" resultMap="lt.velykis.roberta.budget.app.account.AccountRepository.accountMap" />`. I think you can reference it with a fully-qualified name..